lib/factory_girl/registry.rb

```diff
@@ -13,6 +13,10 @@ def clear
       @items.clear
     end
 
+    def remove(name)
+      @items.delete(name.to_sym)
```

Comment:
My question then is, "Why would you as a developer using another factories from another library, attempt to delete something unless you know it's causing problems?" Is there something other than name that you're using to dictate whether something will be deleted, or is there significant churn in the provided Spree/Solidus factories file?

I imagine if you attempt to delete something that's not there, it's probably a line of code you wouldn't want to exist in the codebase, since it arguably has no value.